webapp/components/customTokenDrawer/addPairToHemi.tsx

```diff
@@ -17,19 +17,21 @@ export const AddPairToHemi = function () {
   }
 
   return (
-    <WarningBox
-      heading={t('add-this-pair')}
-      onClose={() => setHideWarning(true)}
-      subheading={t.rich('make-a-request-to-add', {
-        link: (chunk: string) => (
-          <ExternalLink
-            className="text-orange-500 hover:text-orange-700"
-            href="https://github.com/hemilabs/token-list/issues/new"
-          >
-            {chunk}
-          </ExternalLink>
-        ),
-      })}
-    />
+    <div>
```

Comment:
Why are these divs needed?

Reply:
The component itself doesn't have a padding. If I don't add the div, they will inherit the padding defined in `drawer-content`, and it won't look as expected. This div is placed so padding is added  - it works as a container of the component. I wouldn't like to add the padding to the component definition because this padding comes from the Drawer, and it is context dependant. The same applies to the other `div`

Reply:
Note that now the `padding-x` is defined at drawer level for each children ([see here](https://github.com/hemilabs/ui-monorepo/pull/789/files#diff-80f90bcbe6a827d266de5580a576f0f796840f61160c2a547081c3818d32f4a9R1)) without needing for manually apply it for every child. The only 2 exceptions in all drawers, can also be excluded with `skip-parent-padding-x`).